src/monitor/monitor_netlink.c

```diff
@@ -26,6 +26,9 @@
 #include <sys/types.h>
 #include <sys/ioctl.h>
 #define __USE_GNU /* needed for struct ucred */
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE /* needed by musl to enable struct ucred */
+#endif
```

Comment:
We should not define _GNU_SOURCE ourselves.
I would remove define __USE_GNU as well an include "config.h" in the beginning of file.
_GNU_SOURCE is automatically part of config.h if available on system.